ProteoformSuiteGUI/LoadDeconvolutionResults.cs

```diff
@@ -32,6 +32,16 @@ public void LoadDeconvolutionResults_Load(object sender, EventArgs e)
         
         private void btn_neucode_CheckedChanged(object sender, EventArgs e)
         {
+            if (btn_unlabeled.Checked)
+            {
+                ProteoformSweet.run_when_form_loads = false; //if unlabeled, don't run automatically. 
```

Comment:
A more succinct form for setting booleans, for future note:
`ProteoformSweet.run_when_form_loads = !btn_unlabeled.Checked;`